### PR TITLE
Added ability to check property exists in single Card

### PR DESCRIPTION
**Pull request by nkaliev**
In current release user unable to check if card has some properties, cause Card class built on magic properties, that means isset() or property_exists() doesn't work, hitting "missing" properties causes exceptions and there's no way to check property existance.
Added __isset() method to check property exists, both functions now works as required.
Also some fixes in README.md. Replaced some Python code.

### CODE REVIEW

**Comment by skyline94** [comment]
What the hell Travis wants from me?)

File: README.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ NOTE: **All properties are READ-ONLY. You cannot modify them.**
     
 ### Filter Cards via Query Parameters
 
-    $cards = Card::where(set='ktk')->where(subtypes='warrior,human')->all();
+    $cards = Card::where(['set' => 'ktk'])->where(['subtypes' => ['warrior', 'human'])->all();
```
Comment (guiwoda) [inline README.md]: Please add missing closing bracket after `'human']`

File: src/DataBag.php [hunk 2]
```diff
@@ -17,6 +17,10 @@ public function __construct($data = [])
         $this->data = $data;
     }
 
+    public function __isset($name) {
```
Comment (guiwoda) [inline src/DataBag.php]: Code style: braces on new line in methods

**Review by guiwoda — CHANGES_REQUESTED** [review]
@skyline94 thank you for the PR!
Would you please fix these minor things, so I can merge?

**Comment by nkaliev** [comment]
@guiwoda Oh, sorry, my fault. Done!

**Review by guiwoda — APPROVED** [review]

**Comment by guiwoda** [comment]
Awesome, thanks!

**Pull request merged**